osu.Game.Tests/Visual/Multiplayer/TestSceneMatchStartControl.cs

```diff
@@ -68,10 +66,19 @@ private void load(GameHost host, AudioManager audio)
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Size = new Vector2(250, 50),
+                    SelectedItem = { BindTarget = selectedItem }
                 }
             };
         });
 
+        protected override void Update()
+        {
+            base.Update();
+
+            if (RoomJoined)
+                selectedItem.Value = MultiplayerClient.APIRoom?.Playlist.SingleOrDefault(i => i.ID == MultiplayerClient.Room?.Settings.PlaylistItemId);
```

Comment:
This is a bit scary. Might warrant an inline comment explaining why this is needed (ie. what piece of the flow is missing in this test setup which would usually handle this).